dist/index.js

```diff
@@ -0,0 +1,5255 @@
+module.exports =
```

Comment:
It is required, due to https://github.com/Pararius/action-terraform/pull/1/files#diff-fab4d7fb461bc6fbe9587f6c03fff98102b1c744145edcf2a993f2ff7cb05a0dR18

So, yeah, have CI/CD take care of this would be nice for an upcoming iteration.